optaplanner-quarkus-integration/optaplanner-quarkus/drl-integration-test/src/test/java/org/optaplanner/quarkus/drl/it/OptaPlannerTestResourceTest.java

```diff
@@ -33,14 +36,21 @@
 
     @Test
     @Timeout(600)
+    @Disabled("There an issue where enabling ANC twice on same KieBase causes rules not to fire" +
+            " which will cause this test to fail. Enable this test once that issue is resolved.")
```

Comment:
I'd add a pointer to the JIRA in question. Otherwise this will be hard to look up in the future.

Reply:
It did not exist at the time; created it now: https://issues.redhat.com/browse/DROOLS-6336